apstra/two_stage_l3_clos_client.go

```diff
@@ -656,6 +655,11 @@ func (o *TwoStageL3ClosClient) CreateIbaWidget(ctx context.Context, data *IbaWid
 	return id, nil
 }
 
+// UpdateIbaWidget updates an IBA Widget.
+func (o *TwoStageL3ClosClient) UpdateIbaWidget(ctx context.Context, id ObjectId, c *IbaWidgetData) error {
+	return o.client.updateIbaWidget(ctx, o.blueprintId, id, c.raw())
```

Comment:
What do you think about slamming `id` (the widget ID) into the `rawIbaWidget` struct here, and removing `id` from the `updateIbaWidget` signature?

Fewer function arguments *feels* better to me, but I'm not sure it's important.

Reply:
I was following the pattern in the other code.
We can fix it evenly across the board